src/modules/opengl.c

```diff
@@ -7,36 +7,29 @@
 
 void ffPrintOpenGL(FFinstance* instance)
 {
-    FFOpenGLResult result;
-    ffStrbufInit(&result.version);
-    ffStrbufInit(&result.renderer);
-    ffStrbufInit(&result.vendor);
-    ffStrbufInit(&result.slv);
```

Comment:
If the strbufs aren't initialized, `ffDetectOpenGL` will write data in random memory places, which most probably will cause a seg fault

Reply:
The reason that the pipeline doesn't fail is that the GitHub VMs don't have OpenGL support